src/components/icons/SCLogo.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react'
```

Comment:
Why do you need to import `React` here? Looks like we don't use it

Reply:
you're right, I changed ts config and not it works fine without importing React, ty